js/timeout_error.go

```diff
@@ -18,44 +18,41 @@
  *
  */
 
-package lib
+package js
 
 import (
 	"fmt"
 	"time"
 
+	"go.k6.io/k6/errext"
+	"go.k6.io/k6/errext/exitcodes"
 	"go.k6.io/k6/lib/consts"
 )
 
-// TimeoutError is used when somethings timeouts
-type TimeoutError struct {
+// timeoutError is used when somethings times out.
```

Comment:
```suggestion
// timeoutError is used when some operation times out.
```

Reply:
Resolved by the cleanup [force-push](https://github.com/k6io/k6/compare/1d741a854b1b18cd3af99f397d857375a630648c..8771485908bf0be49e0855788f5534c0b96a12d3).